pennylane/tape/qnode.py

```diff
@@ -284,7 +284,12 @@ def _validate_backprop_method(device, interface):
             if interface in backprop_devices:
                 # TODO: need a better way of passing existing device init options
                 # to a new device?
-                device = qml.device(backprop_devices[interface], wires=device.wires, analytic=True)
+                device = qml.device(
+                    backprop_devices[interface],
+                    wires=device.wires,
+                    shots=device.shots,

```

Comment:
Ooh, so this is the fix for #1036. Nice catch (didn't understand how this PR fixed that issue at first)! :octopus:

Reply:
Yep, changing from `default.qubit.autograd` to `default.qubit` in the pytest mark caused the tests to fail due to the same bug!